packages/@o3r/schematics/src/rule-factories/ng-add/dependencies.ts

```diff
@@ -142,6 +152,14 @@ export const getPackageInstallConfig = (packageJsonPath: string, tree: Tree, pro
   };
 };
 
+/**
+ * Replace the caret ranges by tildes range
+ * @param range Range to replace
+ */
+export const enforceTildeRange = (range?: string) => {
```

Comment:
to avoid confusion, can this method have a different name from the flag `enforceTildeRange` ?

Reply:
I take suggestion if you have some, otherwize I let it as it is :)